### ISSUE TITLE
How can i modify the train file to multiple gpu version

### ISSUE THREAD

**Issue by Epiphqny** [LossNAN/I3D-Tensorflow#4]
hello, i used the train_ucf file and modify it to multi-gpu according to your resnet-3d version, but do not success, can you provide a multi-gpu version for this model? thanks.

**Comment by LossNAN** [comment]
you are right, this code just for one gpu, the multi_gpu version will public in two days if you need

**Comment by Epiphqny** [comment]
> you are right, this code just for one gpu, the multi_gpu version will public in two days if you need

Yes, i implemented the multi-gpu version myself but there are still some problems, will be very grateful if you can release your version!

**Comment by Epiphqny** [comment]
@LossNAN OK, thank you very much for your help, i will try it.

**Comment by Epiphqny** [comment]
@LossNAN Can i ask how do you deal with the batch normalization items with saving the model?

**Comment by Epiphqny** [comment]
@LossNAN 已加

**Comment by bhkim1020** [comment]
Hello, I also tried to train multiple cpu and refer your last Dec codes.

I have seen the multi_gpu_train_kinetics_rgb.py file, 
and there seems to be a non_local related code that is not in git. 
I wonder if there was no change in the i3d internals. ex. share variable in i3d network........

**Comment by sanolans** [comment]
Hi @LossNAN, thank you for sharing your work. Do you have the code for training a singular gpu as I am using 1 gpu? Also, while using trying the multi_gpu code, there is a NameError because the learning_rate is not defined, amI doing something wrong? Can you also please please explain the non_local reference in the code?
